### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by arm4b** [bitovi/bitops#289] [Announcements]
## What's Changed
* BitOps rewrite with Python
* New Plugin implementation by @ConnorGraham in https://github.com/bitovi/bitops/pull/55
* Adding manual bump and updating release pipeline conditional by @PhillypHenning in https://github.com/bitovi/bitops/pull/248
* Adjusts alignment and fixes small bug by @WestonVincze in https://github.com/bitovi/bitops/pull/250
* Add GitHub star button to the docs page by @armab in https://github.com/bitovi/bitops/pull/223
* Remove plugin and deployment config from base by @mickmcgrath13 in https://github.com/bitovi/bitops/pull/255
* Remove unused v1 GH job for master branch by @armab in https://github.com/bitovi/bitops/pull/263
* Initial Documentation update DevOps2-90 by @ccapell in https://github.com/bitovi/bitops/pull/264
* Fix `base` image docker tagging by @armab in https://github.com/bitovi/bitops/pull/265
* Updating splash about by @PhillypHenning in https://github.com/bitovi/bitops/pull/273
* Fix missing `ENVIRONMENT` -> `BITOPS_ENVIRONMENT` var changes in the docs by @armab in https://github.com/bitovi/bitops/pull/269
* Fix `deploy_plugins` to exit with non-zero return code on error by @armab in https://github.com/bitovi/bitops/pull/268
* Fixed broken links and examples by @ccapell in https://github.com/bitovi/bitops/pull/272
* Build and deploy `dev` image for omnibus by @armab in https://github.com/bitovi/bitops/pull/267
* Add GH Discussions badge by @armab in https://github.com/bitovi/bitops/pull/274
* `_default` folder override by @PhillypHenning in https://github.com/bitovi/bitops/pull/275
* Add a deprecation warning for `ENVIRONMENT` & env fallback by @armab in https://github.com/bitovi/bitops/pull/277
* Document the stabilized BitOps versioning in the docs by @armab in https://github.com/bitovi/bitops/pull/278
* Remove inexistent ENV vars from the docs by @armab in https://github.com/bitovi/bitops/pull/279
* Development guide fixes by @armab in https://github.com/bitovi/bitops/pull/284
* Various documentation fixes by @armab in https://github.com/bitovi/bitops/pull/281

## New Contributors
* @WestonVincze made their first contribution in https://github.com/bitovi/bitops/pull/250
* @ccapell made their first contribution in https://github.com/bitovi/bitops/pull/264

**Migration Notes**: https://bitovi.github.io/bitops/migration/2.0-migration/
**Full Changelog**: https://github.com/bitovi/bitops/compare/1.9.0...v2.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/bitovi/bitops/releases/tag/v2.0.0'>v2.0.0</a>.</em>

**Comment by mickmcgrath13** [comment]
Super excited for this V2 Release!  A lot of hard work went into it, and it sets a nice foundation for the future!
